internal/helm/helmadapter/helm3_env_service.go

```diff
@@ -224,10 +224,12 @@ func (h helm3EnvService) ListCharts(ctx context.Context, helmEnv helm.HelmEnv, f
 			Name:   filter.RepoFilter(),
 			Charts: make([]interface{}, 0, 0),
 		}
+
+		versions := make([]interface{}, 0, len(chartVersions))
 		for _, chartVersion := range chartVersions {
-			// omg!
-			chartEntry.Charts = append(chartEntry.Charts, []interface{}{chartVersion})
+			chartEntry.Charts = append(versions, chartVersion)
```

Comment:
This doesn't seem right. Shouldn't it be `versions = append(versions, chartVersion)` ?

Reply:
thanks!